### ISSUE TITLE
macOS only

### ISSUE THREAD

**Issue by andypiper** [YuG1224/gh-lgtmoon#2]
pbcopy and pbpaste are Mac commands, you could alias them on Linux to something that does the equivalent, or mark the extension as Mac-only.

**Comment by ItsDrike** [comment]
Indeed, the script is failing on linux with `/home/itsdrike/.local/share/gh/extensions/gh-lgtmoon/gh-lgtmoon: line 4: pbcopy: command not found`.

A linux alternative would be using one of these:
- `xsel` tool: `xsel --clipboard --input`
- `xclip` tool: `xclip -selection clipboard`

It will depend on the distro which of these will actually be available, but usually one of these tools will be installed. Though there are distributions such as Arch Linux, where the user decides what is and what isn't installed, so you can't assume with certainty that it actually will have one of those.

The way I would handle it is to first check for `pbcopy`, if that's missing check for `xsel`, and if that's also not found try `xclip`. In the rare case that none of these would actually be installed, instead of just completely failing with an error message, you should write a warning message like: `No tool found for automatic copying (tried: pbcopy, xsel, xclip)` and then proceed to at least print out the text, so that the user can copy it themselves.
